juego.py

```diff
@@ -67,24 +65,14 @@ def __init__(self, level, fuente):
                   int(sx(1060))]
 
         def rand_generator_x():
-            count_x = 0
-            while count_x < len(list_x):
-                rand_coord_x = random.choice(list_x)
-                count_x += 1
-                if rand_coord_x not in no_repeat_check_x:
-                    no_repeat_check_x.append(rand_coord_x)
-                    return rand_coord_x
-            return int(sx(1000))
+            rand_coord_x = random.choice(list_x)
+            list_x.remove(rand_coord_x)
```

Comment:
Why remove an item you just created?

Reply:
So basically we choose a random position from the list of predefined positions, and return that. Also to make sure that no overlapping with other numbers occur, we remove the selected position from the list.